### PR TITLE
[ticket/15346] Check if extension is enableable during install

### PR DESCRIPTION
**Pull request by rubencm**
PHPBB3-15346

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15346

### CODE REVIEW

File: phpBB/phpbb/install/module/install_finish/task/install_extensions.php [hunk 1]
```diff
@@ -118,6 +118,13 @@ public function run()
 
 			try
 			{
+				$extension = $this->extension_manager->get_extension($ext_name);
+
+				if (!$extension->is_enableable())
+				{
+					continue;
```
Comment (Nicofuma) [inline phpBB/phpbb/install/module/install_finish/task/install_extensions.php]: You should log it I think ($this->iohandler->add_log_message(array('CLI_EXTENSION_ENABLE_FAILURE', $ext_name));)

Comment (rubencm) [reply phpBB/phpbb/install/module/install_finish/task/install_extensions.php]: Yes

**Comment by VSEphpbb** [comment]
Is it reallya failure? IMO the appropriate message would be to say the extension was "skipped"

**Comment by Nicofuma** [comment]
I agree, it's not a failure. I was just copy / pasting the line I was already seeing :p

**Pull request merged**